cub/test/catch2_test_env_launch_helper.h

```diff
@@ -0,0 +1,567 @@
+/******************************************************************************
+ * Copyright (c) 2011-2025, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Question: Should the Copyright start in 2011? Is this correct or a copy&paste error?
Suggestion: use the 2 line SPDX header